components/script/dom/idbrequest.rs

```diff
@@ -129,15 +132,21 @@ impl RequestListener {
                 IdbResult::None => {
                     // no-op
                 },
-                IdbResult::Error(_err) => {
-                    request.set_result(answer.handle());
-                    Self::handle_async_request_error(request, &global);
+                IdbResult::Error(error) => {
```

Comment:
We don't actually construct an IdbResult::Error anywhere, it appears. It's an unused variant; should it be removed?